controllers/basicAuth_test.go

```diff
@@ -24,15 +24,15 @@ func Test_BasicLoginHandler_MalformedRequest(t *testing.T) {
 	userService := mocksUserService.NewUserService(t)
 	sessionService := mocksSessionService.NewSessionService(t)
 
-	controller := controllers.NewBasicAuthentificationController(
+	controller := controllers.NewBasicAuthenticationController(
 		logger,
 		userService,
 		sessionService,
 	)
 	response := httptest.NewRecorder()
 	request := httptest.NewRequest(
 		"POST",
-		"/v1/auth/basic/login",
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.